bench/locli/src/Cardano/Analysis/API/Field.hs

```diff
@@ -1,4 +1,5 @@
-{-# LANGUAGE TypeInType #-}
+{-# LANGUAGE DataKinds #-}
```

Comment:
```suggestion
```

I believe this is redundant? DataKinds is set in the default-extensions in the .cabal file already.

Reply:
I didn't realise DataKinds was going to get added to default-extensions when I added it. I'll keep my hands off to avoid clashes pushing, though.